requirements.txt

```diff
@@ -0,0 +1,5 @@
+pyyaml >= 5.1
```

Comment:
This is presumably entirely wrong?

Reply:
I'm not familiar with the setup of this file whatsoever; `analysis_tools` was used as a base to copy over into this new repo. What should live here instead of this?

Reply:
You need this file if you are building the package and testing it in a GitHub action. If you are depending on afw and the like you won't be able to do that anyway so delete it until you work out what your actual dependencies are.

Reply:
If I remove `requirements.txt` entirely, `mypy` fails its check:
```
Run actions/setup-python@v3
  with:
    python-version: [3](https://github.com/lsst/source_injection/runs/7678233644?check_suite_focus=true#step:3:3).10
    cache: pip
    token: ***
Successfully setup CPython (3.10.[5](https://github.com/lsst/source_injection/runs/7678233644?check_suite_focus=true#step:3:5))
Error: No file in /home/runner/work/source_injection/source_injection matched to [**/requirements.txt], make sure you have checked out the target repository
```

Reply:
Updated to:
```
pandas >= 1.0
numpy >= 1.17
astropy >= 4.0
```
which I believe satisfies the core of this repo for now.